module/x/gravity/keeper/attestation_handler.go

```diff
@@ -9,7 +9,7 @@ import (
 	"github.com/cosmos/gravity-bridge/module/x/gravity/types"
 )
 
-// AttestationHandler processes `observed` Attestations
+// AttestationHandler processes `observed` EthereumEventVoteRecords
 type AttestationHandler struct {
```

Comment:
We should rename this to `EthereumEventVoteHandler` also IDK if this abstraction is useful. Lets just keep an eye on it during the refactor.

Reply:
need to store the votes somewhere, right? Are you think of a design where they just have a "foreign key"?